scripts/classes/app-binary-configurator.js

```diff
@@ -82,7 +84,7 @@ class AppBinaryConfigurator {
           this.publishingProperties = JSON.parse(body);
           resolve();
         } else {
-          reject(`Publishing info download failed with code: ${response.statusCode}`);
+          reject(`Publishing info download failed with code: ${response.statusCode}`.bold.red);
```

Comment:
Do we have a more descriptive error message in the error response that we could print here?